### PR TITLE
CI: prospective fix for the current faillure

### PR DESCRIPTION
**Pull request by ogoffart**
https://github.com/jurplel/install-qt-action/issues/142

### CODE REVIEW

**Comment by ogoffart** [comment]
Did not work :-(

**Comment by ogoffart** [comment]
Although it looks like the faillure is something else now. It just failled to download on one of the job

**Comment by tronical** [comment]
Yes, and the other build_and_that that I think previously failed passed now. So this looks like an improvement to me.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -119,6 +120,7 @@ jobs:
       with:
         version: '5.15.2'
         cached: ${{ steps.cache-qt.outputs.cache-hit }}
+        setup-python: false
```
Comment (tronical) [inline .github/workflows/ci.yaml:123]: I guess this will also do it for Linux, but doesn't seem to work

**Review by tronical — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (ogoffart) [reply .github/workflows/ci.yaml:123]: and it's fine.
In fact i should have added it everywhere since we don't use PySide at all.

**Pull request merged**